fuse_tutorials/src/range_sensor_simulator.cpp

```diff
@@ -190,7 +198,8 @@ nav_msgs::Odometry::ConstPtr robotToOdometry(const Robot& state)
  *
  * The state estimator will not run until it has been sent a starting pose.
  */
-void initializeStateEstimation(const Robot& state)
+void initializeStateEstimation(
+  const Robot& state, rclcpp::node_interfaces::NodeClockInterface::SharedPtr clock_interface=nullptr)
```

Comment:
Same comment about taking a clock instead of a clock interface here.

Reply:
https://github.com/locusrobotics/fuse/pull/283/commits/59d10ddc448f2a21928d5104dabe494e0e37f8e4

Reply:
https://github.com/locusrobotics/fuse/pull/283/commits/bc0d396babf1984836742cb31697d51319f60f2e !